doc/networking.md

```diff
@@ -927,3 +1031,5 @@ currently supported, namely `ipv4` and `ipv6`.
     mapping the low-order 23-bits of the IP address in the low-order 23
     bits of the Ethernet address 01:00:5E:00:00:00.  Meaning, more than
 	one IP multicast group maps to the same MAC multicast group.
+[^4]: A default route with very high admin distance is set to be able to
+    reach a gateway in case no other IP address or route is available.
```

Comment:
Is this reference used anywhere? If it is to be referenced, should we stick to the term "routing preference"?

Reply:
Thanks, went through multiple references of this document. I'll clean it up, making sure we consistently use `routing preference`